thermo/data/co2-thermo.yaml

```diff
@@ -0,0 +1,62 @@
+description: |-
+  CO2 phases based on the equation of state 
+
+cantera-version: 2.5.0a3
```

Comment:
Unless this was converted from a CK file, you don't need to include this.

Reply:
Cleaned up.